src/db/marketplaces/marketplace-events.repository.ts

```diff
@@ -32,16 +32,27 @@ export class MarketplaceEventsRepository extends Repository<MarketplaceEventsEnt
     return savedRecords;
   }
 
```

Comment:
could you change the name, it is not clear why you get be marketpace and Timestamp

Reply:
Fixed